CMakeLists.txt

```diff
@@ -47,4 +47,8 @@ vita_create_vpk(
 	VERSION ${VITA_VERSION}
 	NAME ${VITA_APP_NAME}
 	FILE bin/monorale.bin monorale.bin	
+		 src/sce_sys/icon0.png sce_sys/icon0.png
```

Comment:
without FILE, CMAKE won't recognize.

Reply:
It compiled and included all of the files on my end though?

Reply:
but you should follow coding basis.

Reply:
Alright